libs/ledger-live-common/src/families/bitcoin/hw-getAddress.ts

```diff
@@ -50,38 +37,6 @@ export const getAddressWithBtcInstance = async (
 
   const { bitcoinAddress, publicKey, chainCode } = result;
 
-  if (!skipAppFailSafeCheck) {
```

Comment:
yes, I've also included this in the current PR because I think it is also a part of the code cleanup to remove useless API call. (I include it in another PR because it breaks the new nano app API )
I think it is safe to remove this piece of code here.